trace_decoder/src/typed_mpt.rs

```diff
@@ -126,6 +126,11 @@ impl TrieKey {
     pub fn from_address(address: Address) -> Self {
         Self::from_hash(keccak_hash::keccak(address))
     }
+    pub fn from_slot_position(pos: U256) -> Self {
+        let mut bytes = [0; 32];
+        pos.to_big_endian(&mut bytes);
+        Self::from_hash(keccak_hash::keccak(bytes))
+    }
```

Comment:
@0xaatif this and the getter below have been added for convenience. Happy to remove them if you'd rather not expand the API too much

Reply:
Happy with this :)